package-lock.json

```diff
@@ -1809,9 +1809,9 @@
       }
     },
     "node_modules/@babel/compat-data": {
-      "version": "7.22.20",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.22.20.tgz",
-      "integrity": "sha512-BQYjKbpXjoXwFW5jGqiizJQQT/aC7pFm9Ok1OWssonuguICi264lbgMzRp2ZMmRSlfkX6DsWDDcsrctK8Rwfiw==",
+      "version": "7.23.2",
```

Comment:
Looks like mobile view in storybook is broken? Can't select a story in the sidebar and switch back to see the story, it always goes back to the anchor story.

![mobileview](https://github.com/ni/nimble/assets/1588923/946ddd62-a245-4f75-92e0-be91164ba531)

Reply:
Good catch! But I see the same thing in https://nimble.ni.dev/storybook/ so I think it's unrelated to this upgrade. If you do too, let's file a bug to Nimble and we can see if it's our configuration or something wrong with SB.

Reply:
Ahh yea, shoulda checked, yea I see it there to. Made an issue and will approve this one: https://github.com/ni/nimble/issues/1639